website/docs/reference/warehouse-profiles/layer-profile.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Layer Profile"
+---
+
+:::info Vendor-supported plugin
+
+Some [core functionality](https://github.com/pingcap/dbt-tidb/blob/main/README.md#supported-features) may be limited. 
+If you're interested in contributing, check out the source code repository listed below.
+
+:::
+
+## Overview of dbt-layer
+**Maintained by:** Layer      
+**Authors:** Mehmet Ecevit  
+**Source:** [Github](https://github.com/layerai/dbt-layer)  
+**Core version:** v0.1 and newer      
+**dbt Cloud:** Not Supported      
+
+To immediately start using Layer inside your dbt DAG, install the Layer dbt Adapter for BigQuery.
+
+```
+pip install dbt-layer-bigquery -U
+```
+
+You don't need to install dbt separately. Installing `dbt-layer` will also install `dbt-core`
+
+
+### Connecting to Layer with **dbt-layer**
+
+#### Username / password authentication
+
+Add the Layer dbt Adapter to your BigQuery profile. An example profile:
```

Comment:
might also be worth linking to the [BigQuery profile set up page](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile)? bc all methods would be supported? this is done for dbt-synapse profile set up, where it just points users to dbt-sqlserver's profile set up, because the authentication is the same/inherited

Reply:
Good idea, addressed here:
https://github.com/dbt-labs/docs.getdbt.com/pull/1542/commits/fed2b1b33e870ca6870a5ad340d591940886579d#diff-46c568a2d197c0cde149cba590515c192a792abc2d8b27bbbfc1a874b285927aR34